Server/Quality_of_Life_changer.WebApi/Controllers/UserProfileController.cs

```diff
@@ -36,6 +44,27 @@ public async Task<IActionResult> AddCalendar([FromBody] CalendarModel model, str
         return Ok(response);
     }
 
+    [HttpGet("calendars")]
+    [Authorize]
+    public async Task<IActionResult> GetCalendars(string userId)
+    {
+        if (!IsValidId(userId, HttpContext))
+        {
+            throw new ValidationException("User id from url not equals id from token");
```

Comment:
better return something like 403 status code in such cases. it should be another exception type